sct-commons/src/main/java/org/lfenergy/compas/sct/commons/scl/ied/LDeviceAdapter.java

```diff
@@ -137,75 +187,81 @@ public String getLdName() {
      * Checks if LDevice has an LN0 node
+     *
      * @return true if lDevice has a LN0 node, false otherwise
      */
-    public boolean hasLN0(){
+    public boolean hasLN0() {
         return currentElem.isSetLN0();
     }
 
     /**
      * Gets current LDevice LNodes (except LN0)
+     *
      * @return list of <em>LNAdapter</em> object
      */
-    public List<LNAdapter> getLNAdapters(){
+    public List<LNAdapter> getLNAdapters() {
         return currentElem.getLN()
                 .stream()
-                .map(tln -> new LNAdapter(this,tln))
+                .map(tln -> new LNAdapter(this, tln))
                 .toList();
     }
 
     /**
      * Gets specific LNode from current LDevice
+     *
      * @param lnClass LNode lnClass value
-     * @param lnInst LNode lnInst value
-     * @param prefix LNode prefix value
+     * @param lnInst  LNode lnInst value
+     * @param prefix  LNode prefix value
      * @return <em>LNAdapter</em> object
      * @throws ScdException thros when specified LNode not found in current IED
      */
     public LNAdapter getLNAdapter(String lnClass, String lnInst, String prefix) throws ScdException {
         return findLnAdapter(lnClass, lnInst, prefix)
                 .orElseThrow(
-                        ()-> new ScdException(
-                            String.format(
-                                "LDevice [%s] has no LN [%s,%s,%s]", currentElem.getInst(),lnClass,lnInst,prefix)
-                            )
-                        );
+                        () -> new ScdException(
+                                String.format(
+                                        "LDevice [%s] has no LN [%s,%s,%s]", currentElem.getInst(), lnClass, lnInst, prefix)
+                        )
+                );
 
     }
 
     /**
      * Find a specific LN from current LDevice
+     *
      * @param lnClass LNode lnClass value
-     * @param lnInst LNode lnInst value
-     * @param prefix LNode prefix value
+     * @param lnInst  LNode lnInst value
+     * @param prefix  LNode prefix value
      * @return <em>LNAdapter</em> object
      * @throws ScdException thros when specified LNode not found in current IED
      */
     public Optional<LNAdapter> findLnAdapter(String lnClass, String lnInst, String prefix) {
-        if (!currentElem.isSetLN()){
+        if (!currentElem.isSetLN()) {
             return Optional.empty();
         }
         return currentElem.getLN()
-            .stream()
-            .filter(tln -> Utils.lnClassEquals(tln.getLnClass(), lnClass)
-                && tln.getInst().equals(lnInst)
-                && Utils.equalsOrBothBlank(prefix, tln.getPrefix()))
-            .map(tln -> new LNAdapter(this, tln))
-            .findFirst();
+                .stream()
+                .filter(tln -> Utils.lnClassEquals(tln.getLnClass(), lnClass)
+                        && tln.getInst().equals(lnInst)
+                        && Utils.equalsOrBothBlank(prefix, tln.getPrefix()))
```

Comment:
`Utils.equalsOrBothBlank` is used more than 20 times in Compas, so I think its worth making a method for that. And I like methods that explicitly says what there doing, it feels more Clean Code, even if the implementation is short.
Your code works perfectly like `Utils.equalsOrBothBlank` by the way, but It confused me that `StringUtils.trimToEmpty(null)` returns `""` instead of `null`.